bin/server/cmd/root.go

```diff
@@ -364,6 +368,8 @@ func (m *Main) Close() error {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
 
+	m.l.Info("Trying to cancel main...")
```

Comment:
"cancel main" is from the programming perspective.

Since this surfaces to the user, better to alter it to say something familiar to the user.

Reply:
Done